runtime/parachains/src/inclusion.rs

```diff
@@ -255,20 +262,24 @@ impl<T: Trait> Module<T> {
 		let threshold = availability_threshold(validators.len());
 
 		let mut freed_cores = Vec::with_capacity(n_bits);
-		for (para_id, pending_availability) in assigned_paras_record.into_iter()
+		for (para_id, pending_availability, commitments) in assigned_paras_record.into_iter()
 			.filter_map(|x| x)
-			.filter_map(|(id, p)| p.map(|p| (id, p)))
+			.filter_map(|(id, p, c)| p.and_then(|p| c.map(|c| (id, p, c))))
```

Comment:
this could be
```rust
.filter_map(|(id, p, c)| p.map(|p| (id, p, c.expect("commitments are not None when pending_availability is Some; qed"))))

Reply:
I think this is outdated by my prior comment.